apps/projects/management/commands/util.py

```diff
@@ -0,0 +1,101 @@
+import pytz
+import requests
+
+from django.core.management.base import CommandError
+from django.utils import timezone
+
+from adhocracy4.modules.models import Module
+from adhocracy4.phases.models import Phase
+from adhocracy4.projects.models import Project
+
+
+def a3_login(url, headers, username, password):
+    login_url = url + 'login_username'
+    res = requests.post(
+        login_url,
+        json={'name': username, 'password': password}
+    )
+    if res.status_code != requests.codes.ok:
+        raise CommandError('API user authentication failed.')
+    headers['X-User-Token'] = res.json()['user_token']
```

Comment:
I think it would improve readability a great deal if this returned the token instead of setting it directly.